tools/cd_scripts/package_gcsfuse.sh

```diff
@@ -76,44 +71,27 @@ sudo apt-get update
 echo Install docker
 sudo apt install apt-transport-https ca-certificates curl software-properties-common -y
 curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
-sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu focal stable"
+sudo add-apt-repository "deb [arch=${architecture}] https://download.docker.com/linux/ubuntu focal stable"
 apt-cache policy docker-ce
 sudo apt install docker-ce -y
 echo Install git
 sudo apt-get install git -y
 # It is require for multi-arch support
 sudo apt-get install qemu-user-static binfmt-support
-git clone https://github.com/GoogleCloudPlatform/gcsfuse.git
-git checkout "v$RELEASE_VERSION_TAG"
+git clone https://github.com/Tulsishah/gcsfuse
 cd gcsfuse/tools/package_gcsfuse_docker/
 # Setting set +e to capture error output in log file and send it on the bucket.
 set +e
 exit_code=0
-echo "Building docker for amd64 ..."
-sudo docker buildx build --load . -t gcsfuse-release-amd64:"$RELEASE_VERSION_TAG" --build-arg GCSFUSE_VERSION="$RELEASE_VERSION" --build-arg ARCHITECTURE=amd64 --build-arg BRANCH_NAME="v$RELEASE_VERSION_TAG" --platform=linux/amd64 &> docker_amd64.log &
-pid1=$!
-echo "Building docker for arm64 ..."
-# It is necessary for cross-platform image building because it creates a builder instance that is capable of
-# building images for multiple architectures.
-sudo docker buildx create --name mybuilder --bootstrap --use
-sudo docker buildx build --load . -t gcsfuse-release-arm64:"$RELEASE_VERSION_TAG" --build-arg GCSFUSE_VERSION="$RELEASE_VERSION" --build-arg ARCHITECTURE=arm64 --build-arg BRANCH_NAME="v$RELEASE_VERSION_TAG" --platform=linux/arm64 &> docker_arm64.log &
-pid2=$!
-echo "Waiting for both builds to complete ..."
-wait $pid1
-check_docker_build_failure
-gsutil cp docker_amd64.log gs://"$UPLOAD_BUCKET"/v"$RELEASE_VERSION"/
-wait $pid2
-check_docker_build_failure
-gsutil cp docker_arm64.log gs://"$UPLOAD_BUCKET"/v"$RELEASE_VERSION"/
-
-# Exit if any of the build fails.
-exit_in_failure
+echo "Building docker for ${architecture} ..."
+sudo docker buildx build --load . -t gcsfuse-release-${architecture}:"$RELEASE_VERSION_TAG" --build-arg GCSFUSE_VERSION="$RELEASE_VERSION" --build-arg ARCHITECTURE=${architecture} --build-arg BRANCH_NAME="v$RELEASE_VERSION_TAG" &> docker_${architecture}.log
```

Comment:
is `--platform=linux/${architecture}` not required?

Reply:
No